build.proj

```diff
@@ -71,6 +71,18 @@
   <Import Project="dir.targets" />
   <Import Project="dir.traversal.targets" />
   <Import Project="$(ToolsDir)clean.targets" />
+  
+  <Target Name="CollectTemplateFiles" BeforeTargets="UpdateDependencies" >
```

Comment:
Now that I don't need Remove which only works for dynamic itemgroups I don't need a separate target.